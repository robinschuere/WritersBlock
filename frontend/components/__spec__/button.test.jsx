import React from 'react';
import renderer from 'react-test-renderer';
import Button from '../generic/button';

test('Button', () => {
  const onHandleClick = (event) => {
    expect(event).toEqual('');
  };

  const component = renderer.create(
    <Button onClick={onHandleClick}>TEXT</Button>,
  );

  const tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});
