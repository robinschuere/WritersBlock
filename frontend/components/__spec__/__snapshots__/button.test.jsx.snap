// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Button 1`] = `
<button
  className="btn btn-lg btn-default "
  onClick={[Function]}
  type="button"
>
  
  TEXT
</button>
`;
